arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -0,0 +1,2508 @@
+    "Bitness": {
+      "description": "Specifies the bitness of a machine or process.",
+      "type": "string",
+      "enum": [
+        "32bit",
+        "64bit"
+      ],
+      "x-ms-enum": {
+        "name": "Bitness",
+        "modelAsString": false
+      },
+      "x-ms-external": true
+    },
+    "OperatingSystemFamily": {
+      "description": "Specifies the operating system family, e.g., Linux, Windows, etc.",
+      "type": "string",
+      "enum": [
+        "unknown",
+        "windows",
+        "linux",
+        "solaris",
+        "aix"
+      ],
+      "x-ms-enum": {
+        "name": "OperatingSystemFamily",
+        "modelAsString": false
+      }
+    },
+    "MachineRebootStatus": {
+      "description": "Specifies if the machine has been rebooted since the installation of the dependency agent.",
+      "type": "string",
+      "enum": [
+        "unknown",
+        "rebooted",
+        "notRebooted"
+      ],
+      "x-ms-enum": {
+        "name": "MachineRebootStatus",
+        "modelAsString": false
+      }
+    },
+    "Accuracy": {
+      "description": "Specifies the accuracy of a computation.",
+      "type": "string",
+      "enum": [
+        "actual",
+        "estimated"
+      ],
+      "x-ms-enum": {
+        "name": "Accuracy",
+        "modelAsString": false
+      }
+    },
+    "VirtualMachineType": {
+      "description": "Specifies the virtualization type of a machine.",
+      "type": "string",
+      "enum": [
+        "unknown",
+        "hyperv",
+        "ldom",
+        "lpar",
+        "vmware",
+        "virtualPc",
+        "xen"
+      ],
+      "x-ms-enum": {
+        "name": "VirtualMachineType",
+        "modelAsString": false
+      }
+    },
+    "HypervisorType": {
+      "description": "Specifies the hypervisor type of a machine.",
+      "type": "string",
+      "enum": [
+        "unknown",
+        "hyperv"
+      ],
+      "x-ms-enum": {
+        "name": "HypervisorType",
+        "modelAsString": false
+      }
+    },
+    "ConnectionFailureState": {
+      "description": "Connection failure state:\n * ```ok``` indicates no failures\n * ```failed``` indicates only failures\n * ```mixed``` indicates both failures and successes ",
+      "type": "string",
+      "enum": [
+        "ok",
+        "failed",
+        "mixed"
+      ],
+      "x-ms-enum": {
+        "name": "ConnectionFailureState",
+        "modelAsString": false
+      }
+    },
+    "SummaryProperties": {
+      "type": "object",
+      "description": "Base for all summaries. ",
+      "properties": {
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Summary internval start time."
+        },
+        "endTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Summary internval end time."
+        }
+      },
+      "required": [
+        "startTime",
+        "endTime"
+      ]
+    },
+    "MachineCountsBy
```

Comment:
please add `"x-ms-parameter-location": "method"` so that the parameter is not tied to the service client